addons/apecloud-mysql-cluster/templates/etcd-service-descriptor.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if eq .Values.etcd.serviceReference.cluster ""}}
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ServiceDescriptor
+metadata:
+  name: {{ include "kblib.clusterName" . }}-etcd-descriptor
+  namespace: {{ .Release.Namespace }}
```

Comment:
the namespace of SD is defined explicitly. 
this could be a different value used in `cluster.componentSpecs.serviceRef`
```
namespace: {{ .Values.etcd.serviceReference.namespace | default .Release.Namespace }}
```